content/content.json

```diff
@@ -125,6 +125,13 @@
       "title": "How to Set Up Goose on Windows",
       "link": "https://dev.to/lymah/how-to-set-up-goose-on-windows-29dd",
       "contributor": "Lymah123"
+    },
+    {
+      "type": "short",
+      "thumbnail": "./img/Short About Using .goosehints Files with Goose.png",
```

Comment:
same with this image title..could you please rename to short-about-goosehints-file.png? Thank you!!!